doc/api/vm.md

```diff
@@ -125,6 +125,10 @@ console.log(util.inspect(sandbox));
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
+*Note*: Using the `timeout` or `breakOnSigint` options will result in new
+event loops and corresponding threads being started, which have a non-zero
```

Comment:
@aqrln The `timeout` option will create *both* a thread and a new event loop. ;) (I would actually assume spinning up the thread is much more costly – basically, what I’m trying to say is that both will allocate and consume new resources.)